src/Workspaces/Core/Portable/Utilities/TaskExtensions.cs

```diff
@@ -264,6 +278,8 @@ public static T WaitAndGetResult_CanCallOnBackground<T>(this Task<T> task, Cance
             TaskContinuationOptions continuationOptions,
             TaskScheduler scheduler)
         {
+            Contract.ThrowIfNull(continuationFunction, nameof(continuationFunction));
+
             return task.SafeContinueWithFromAsync<TResult>(
                 (Task antecedent) => continuationFunction((Task<TInput>)antecedent), cancellationToken, continuationOptions, scheduler);
```

Comment:
:memo: The bug I mentioned above is a crash report for a `NullReferenceException` pointing to this callback, meaning one of the following occurred:

1. `continuationFunction` was null.
2. `continuationFunction` was inlined, and the implementation of it threw a `NullReferenceException`.